csl-data.json

```diff
@@ -1,447 +1,440 @@
+                },
+                "recipient": {
+                    "$ref": "#/definitions/nameDefinition"
+                },
+                "reviewed-author": {
+                    "$ref": "#/definitions/nameDefinition"
+                },
+                "translator": {
+                    "$ref": "#/definitions/nameDefinition"
+                }
+            }
+        },
+        "numberFields": {
+            "properties": {
+                "chapter-number": {
+                    "$ref": "#/definitions/numberDefinition"
+                },
+                "collection-number": {
+                    "$ref": "#/definitions/numberDefinition"
+                },
+                "edition": {
+                    "$ref": "#/definitions/numberDefinition"
+                },
+                "issue": {
+                    "$ref": "#/definitions/numberDefinition"
+                },
+                "number": {
+                    "$ref": "#/definitions/numberDefinition"
+                },
+                "number-of-pages": {
+                    "$ref": "#/definitions/numberDefinition"
+                },
+                "number-of-volumes": {
+                    "$ref": "#/definitions/numberDefinition"
+                },
+                "volume": {
+                    "$ref": "#/definitions/numberDefinition"
+                }
+            }
+        },
+        "stringFields": {
+            "properties": {
+                "abstract": {
+                    "type": "string"
+                },
+                "annote": {
+                    "type": "string"
+                },
+                "archive": {
+                    "type": "string"
+                },
+                "archive_location": {
+                    "type": "string"
+                },
+                "archive-place": {
+                    "type": "string"
+                },
+                "authority": {
+                    "type": "string"
+                },
+                "call-number": {
+                    "type": "string"
+                },
+                "citation-label": {
+                    "type": "string"
+                },
+                "citation-number": {
+                    "type": "string"
+                },
+                "collection-title": {
+                    "type": "string"
+                },
+                "container-title": {
+                    "type": "string"
+                },
+                "container-title-short": {
+                    "type": "string"
+                },
+                "dimensions": {
+                    "type": "string"
+                },
+                "DOI": {
+                    "type": "string"
+                },
+                "event": {
+                    "type": "string"
+                },
+                "event-place": {
+                    "type": "string"
+                },
+                "first-reference-note-number": {
```

Comment:
This is a processor-generated value that depends on the position of references in a document. Any value set on input will be ignored.

Reply:
So can this be removed?